src/lib/proof-system/zkprogram.ts

```diff
@@ -236,6 +345,22 @@ class DynamicProof<Input, Output> extends ProofBase<Input, Output> {
 
   private static memoizedCounter: number | undefined;
 
+  /**
+   * As the name indicates, feature flags are features of the proof system.
+   *
+   * If we want to side load proofs and verification keys, we first have to tell Pickles what _shape_ of proofs it should expect.
+   *
+   * For example, if we want to side load proofs that use foreign field arithmetic custom gates, we have to make Pickles aware of that by defining
+   * these custom gates.
+   *
+   * _Note:_ Only proofs that use the exact same composition of custom gates which were expected by Pickles can be verified using side loading.
+   * If you want to verify _any_ proof, no matter what custom gates it uses, you can use {@link FeatureFlags.allMaybe}. Please note that this might incur a small overhead.
+   *
+   * You can also toggle specific feature flags manually by specifying them here.
+   * Alternatively, you can use {@FeatureFlags.fromZkProgram} to compute the set of feature flags that are compatible with a given program.
+   */
+  static featureFlags: FeatureFlags = FeatureFlags.allMaybe;
```

Comment:
isn't changing the default from `None` to `Maybe` a breaking change?

I do agree that maybe makes sense as the default

Reply:
Yea it does, sadly.. I'll set it to `none` for now to maintain backwards compatibility :/ The good thing is developers will notice that they need to change something when proofs with different feature flags wont verify